gdal/swig/python/gdal-utils/osgeo_utils/auxiliary/util.py

```diff
@@ -312,10 +312,8 @@ def get_raster_minimum(filename_or_ds: PathOrDS, bnd_index: Optional[int] = 1):
 def get_raster_min_max(filename_or_ds: PathOrDS, bnd_index: int = 1):
     with OpenDS(filename_or_ds) as ds:
         bnd = ds.GetRasterBand(bnd_index)
-        bnd.ComputeStatistics(0)
-        min_val = bnd.GetMinimum()
-        max_val = bnd.GetMaximum()
-        return min_val, max_val
+        min_max = bnd.ComputeRasterMinMax()
```

Comment:
note: by default ComputeRasterMinMax() will recompute the statistics and not use possibly stored ones. if passing approx_ok = 1, then stored ones could be used

Reply:
Thanks, I've added this parameter.